server/database/fake_data/comment.js

```diff
@@ -0,0 +1,48 @@
+const Comment = require('./../models/comment');
+const Technology = require('./../models/technology');
+const User = require('./../models/user');
+
+const insertComments = async () => {
+  // get 'express' and 'Mongo' Technology
+  const technologies = await Technology.find({});
+  const express = technologies[0];
+  const mongodb = technologies[1];
+
+  // get user
+  const users = await User.find();
+  const RamyAlshurafa = users[0];
+  const FarahZaqout = users[2];
+  const HemaSAli = users[6];
+  const MohammedYehia = users[8];
+
+  const comments = [
+    {
+      description: 'Express learning goals look good',
+      user: MohammedYehia,
```

Comment:
This should be a userid. The built in validation should throw an error, but it does not. 

We are storing the entire user object and the schema requires an object id...and somehow it still passes the test.
Anyway, should be refactored.

Reply:
aha, i should have paid attention to the tests.

aproved the pr.